modules/system/traits/ConfigMaker.php

```diff
@@ -1,5 +1,6 @@
 <?php namespace System\Traits;
 
+use Illuminate\Support\Facades\Config;
```

Comment:
why not just `use Config;`?

Reply:
Oops, there's something screwy with my IntelliJ. I'll fix it.